target/linux/ramips/dts/mt7621_ubnt_unifi-flexhd.dts

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_ubnt_unifi.dtsi"
+
+/ {
+	compatible = "ubnt,unifi-flexhd", "mediatek,mt7621-soc";
+	model = "Ubiquiti UniFi FlexHD";
+
+	aliases {
+		/delete-property/ led-boot;
+		/delete-property/ led-failsafe;
+		/delete-property/ led-running;
+		/delete-property/ led-upgrade;
+	};
+
+	leds {
+		/delete-node/ dome_blue;
+		/delete-node/ dome_white;
+	};
```

Comment:
If you need to delete that much from the DTSI, I think it would be better to not use it and just copy the few remaining statements here.

Reply:
I have several commits that are not part of this pull request that split the LED definition to separate "ledgpio" and "ledbar" feature .dtsi files, so that each device can include the correct definition.

Reply:
See https://github.com/swegener/openwrt/commit/91baae0bdc22084bf36e53af508a9830696f76de for splitting the ledgpio .dtsi to remove the delete-* operations from the flexhd .dts.

Reply:
I do not see much point in having a myriad of micro-DTSI files, particularly for LEDs and buttons which have tiny changes of individual GPIO numbers and high/low states for single LED frequently. Please just copy the needed statements from the DTSI and remove the reference as I requested. If you insist, we may discuss the micro-DTSI files in a separate PR, but be aware that I will probably reject it there, too.

Reply:
I've merged all required definitions into the device dts.